tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,15 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+        it("should cache readFileSync on same file paths", () => {
+            const readFileSyncCount = sinon.spy(fs, "readFileSync");
+            const ignoredPaths = new IgnoredPaths({ ignore: true, cwd: getFixturePath() });
+
+            assert.isTrue(ignoredPaths.contains(getFixturePath(".eslintignore")));
```

Comment:
I might be missing something, but it seems like this is only called once, so it's not testing that the cache works. Shouldn't it be called twice instead?